students/17th/team3/jaeyiahn/user/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class User(models.Model):
+    username   = models.CharField(max_length=45, null=True)
+    password   = models.CharField(max_length=500)
+    phone      = models.CharField(max_length=45, null=True)
+    email      = models.EmailField(max_length=100)
```

Comment:
```suggestion
    username   = models.CharField(max_length=45, null=True)
    phone      = models.CharField(max_length=45, null=True)
    email      = models.EmailField(max_length=100)
```
요 세개의 필드는 유일성이 필요하지 않을까요? unique 옵션을 적용하시면 좋을것 같습니다.

Reply:
넵 적용하겠습니다